### PR TITLE
Fix/117421 cannot select start times

### PR DESCRIPTION
**Pull request by nealfennimore**
https://github.com/moderntribe/events-gutenberg/pull/463 should be reviewed before this

https://central.tri.be/issues/117421

https://www.useloom.com/share/028c4a3c3b3d493ebc35f8501d45a8e5

Commits pertinent to this ticket:
- https://github.com/moderntribe/events-gutenberg/commit/a739270b6a82931d79d78641635ced2f6cc69dd8
- https://github.com/moderntribe/events-gutenberg/commit/205e890322ed1b7b07ce377fca3710d0b23852a0

### CODE REVIEW

**Comment by paulmskim** [comment]
Hey @nealfennimore  want to merge in `task/recurrence` back to your feature branch?

File: plugins/events-pro/src/modules/elements/from-time-range-picker/template.js [hunk 1]
```diff
@@ -49,16 +49,6 @@ const FromTimeRangePicker = ( {
 			showAllDay: true,
 			allDay: isAllDay,
 		};
-
-		if ( ! isMultiDay ) {
-			const max = fromSeconds(
-				toSeconds( endTime, TIME_FORMAT_HH_MM ) - MINUTE_IN_SECONDS,
-				TIME_FORMAT_HH_MM,
-			);
-			props.end = momentUtil.roundTime( moment( max, 'HH:mm' ) ).format( 'HH:mm' );
```
Comment (nealfennimore) [inline plugins/events-pro/src/modules/elements/from-time-range-picker/template.js]: @paulmskim 	I don't think I broke anything by removing the `end` here. User's should be able to select any time now and have it update, but let me know if I missed something

File: plugins/events-pro/src/modules/elements/from-time-range-picker/template.js [hunk 2]
```diff
@@ -73,26 +63,6 @@ const FromTimeRangePicker = ( {
 			showAllDay: true,
 			allDay: isAllDay,
 		};
-
-		if ( ! isMultiDay ) {
-			const startTimeSeconds = toSeconds( startTime, TIME_FORMAT_HH_MM );
-			// if the start time has less than half an hour left in the day
-			if ( startTimeSeconds + HALF_HOUR_IN_SECONDS >= DAY_IN_SECONDS ) {
-				props.start = END_OF_DAY;
```
Comment (nealfennimore) [inline plugins/events-pro/src/modules/elements/from-time-range-picker/template.js]: @paulmskim 	Similar comment here as the `end`. Let me know if I missed something

File: plugins/events-pro/src/modules/elements/from-time-range-picker/template.js (hunk 1 above)
Comment (paulmskim) [reply plugins/events-pro/src/modules/elements/from-time-range-picker/template.js]: That's right, removing this should be fine.

File: plugins/events-pro/src/modules/elements/from-time-range-picker/template.js (hunk 2 above)
Comment (paulmskim) [reply plugins/events-pro/src/modules/elements/from-time-range-picker/template.js]: Same as above here.

**Pull request merged**